src/app/work-item/work-item-board/work-item-board.component.scss

```diff
@@ -57,6 +57,9 @@
               justify-content: space-between;
             }
           }//boardcard ended
+          .board-card-selected {
+            border: 3px solid #bee1f4;
```

Comment:
Thanks @joshuawilson and @mindreeper2420 . Done in https://github.com/fabric8io/fabric8-planner/pull/1552/commits/63cb932d186a2ddae16f89808cea0b384928cfc0